debian/Makefile

```diff
@@ -24,32 +24,36 @@ CONSOLEUIDESKTOPDEST:=$(DESTDIR)/usr/share/applications/ckan-consoleui.desktop
 VERSION:=$(shell egrep '^\s*\#\#\s+v.*$$' $(CHANGELOGSRC) | head -1 | sed -e 's/^\s*\#\#\s\+v//' )
 DEB:=../_build/deb/ckan_$(VERSION)_all.deb
 
-APTREPO:=../_build/deb/apt-repo
-REPODEB:=$(APTREPO)/ckan_$(VERSION)_all.deb
-PACKAGES:=$(APTREPO)/Packages.gz
+# Files for /deb/ (the .deb file)
+APT_REPO_ROOT:=../build/deb/apt-repo-root
```

Comment:
```suggestion
APT_REPO_ROOT:=../_build/deb/apt-repo-root
```

Reply:
Oops, thanks. Obviously I'm still not back in Linux yet or I would have been able to test this myself.

Reply:
Builds fine now. I'm wondering why `dpkg-scanpackages` prepends `./`, looking at the Packages.gz of Mono, it is not.
But I hope apt can deal with that.

Reply:
I assume that's because Mono uses a `pool` directory as some of the APT repo guides suggest.

Reply:
Hm yeah, it might not prepend it if the path starts with a directory. Or they are using a different too to generate the Packages file.

Let's give it another shot!